src/linux/resources/jd-gui.desktop

```diff
@@ -2,6 +2,7 @@
 Comment=Java Decompiler JD-GUI
 Terminal=false
 Name=JD-GUI
-Exec=java -jar /opt/jd-gui/jd-gui.jar
+Exec=java -jar /opt/jd-gui/jd-gui.jar %F
 Type=Application
 Icon=jd-gui
+MimeType=application/java-archive;application/java-vm
```

Comment:
According to an [original document by Sun](https://docs.oracle.com/cd/E19146-01/819-2630/abumo/index.html).  The mimetype `application/java-vm` was reserved just for `*.class` files.  Therefore, this should read:

```desktop
MimeType=application/java-archive
```